VMS/Pages/ActivityLogPage/index.razor

```diff
@@ -0,0 +1,25 @@
+﻿@attribute [Route(Routes.ActivityLogPage)]
```

Comment:
Please rename file to Index.razor, not index.razor. Failed to build, "Component 'index' starts with a lowercase character. Component names cannot start with a lowercase character."